pkg/webhooks/namespace/namespace.go

```diff
@@ -150,8 +152,29 @@ func (s *NamespaceWebhook) authorized(request admissionctl.Request) admissionctl
 			ret.UID = request.AdmissionRequest.UID
 			return ret
 		}
-		log.Info("Non-admin access attempt to privileged namespace", "request", request.AdmissionRequest)
-		ret = admissionctl.Denied(fmt.Sprintf("Prevented from creating or changing a potentially harmful Namespace. Customer namespaces should NOT match this regular expressions: %s", privilegedNamespace))
+		log.Info("Non-admin attempted to access a privileged namespace (eg matching this regex)", "regex", privilegedNamespace, "request", request.AdmissionRequest)
+		ret = admissionctl.Denied(fmt.Sprintf("Prevented from accessing Red Hat managed namespaces. Customer workloads should be placed in customer namespaces, and should not match this regex: %s", privilegedNamespace))
```

Comment:
```suggestion
		ret = admissionctl.Denied(fmt.Sprintf("Prevented from accessing Red Hat managed namespaces. Customer workloads should be placed in customer namespaces, and should not match this regular expression: %s", privilegedNamespace))
```

To be consistent with "regular expression" in the other hooks, as opposed to "regex".

Reply:
done :)